docs/combine.rst

```diff
@@ -0,0 +1,105 @@
+combine
+=======
+
+.. dfhack-tool::
+    :summary: Combine stacks of food and plant from one or every stockpile.
+    :tags: fort productivity stockpiles items plants
+
+
+Usage
+-----
+
+::
+
+    combine [help] [<mode opts>] [<stockpile opts>] [<stack opts>] [<output opts>]
+
+
+Examples
+--------
+``combine``
+    Displays help
+``combine --preview --stockpile=all``
+    Preview stack changes for all types in all stockpiles.
+``combine --preview --stockpile=all --max=500``
+    Preview stacks changes with max of 500 per stack for all stockpiles.
+``combine --preview --types=meat,plant``
+    Preview meat and plant stacks changes in all stockpiles.
+``combine --merge --stockpile=all --max=500``
+    Merge stacks for all stockpiles with max of 500 per stack.
+``combine --merge --stockpile=here``
+    Merge stacks for stockpile under cursor; enable cursor through DF settings.
+``combine --merge --stockpile=<num> --types=meat,plant``
+    Merge meat stacks for stockpile id <num>.
```

Comment:
it feels a bit off to require an option in order for the script to do anything. to bring this script API in line with most other scripts, I suggest removing the `--merge` option and converting the `--stockpile` option into a command. e.g.:
- `combine all`: common case, combine everything in all stockpiles
- `combine --dry-run here`: preview changes for the selected stockpile

Reply:
okay, all and here can be the commands. And dry-run will be option.